pkg/aws/policies.go

```diff
@@ -289,7 +299,7 @@ func (c *awsClient) EnsurePolicy(policyArn string, document string,
 		return policyArn, err
 	}
 
-	if !isCompatible {
+	if !isCompatible || force {
```

Comment:
In the case of `force==true` can we skip the compatible check on line `297`?
As we are going to ignore the result anyway.

Reply:
Yes, just thought that this way would change the code less. But I can do that so we have less calls

Reply:
fixed